unearthed-arcana/2017/20170206.xml

```diff
@@ -326,6 +326,11 @@
           <stat name="HP" value="1" level="18" />
```

Comment:
You can also remove all those hp stats and just have: `<stat name="hp" value="level:sorcerer" />`

Reply:
Great suggestion! I made that change.